### ISSUE TITLE
swiping UIPageViewController does not work

### ISSUE THREAD

**Issue by ohadnav** [kif-framework/KIF#1014]
I fail to use `tester().swipeView(withAccessibilityLabel: "label", in: .up)`
for nested view controllers. Any solutions are welcome

**Comment by justinseanmartin** [comment]
Your scenario is that you have a UIScrollView within a UIScrollView? Is the inner UIScrollView the one with an accessibilityLabel of "label" in your scenario? 

If you can add a test scenario to the KIF test app's ScrollView scene and add a test case that tries to exercise your behavior, we can take a look and help see what is going wrong.

**Comment by ohadnav** [comment]
Actually the problem is more delicate. I cannot reproduce it easily.
